exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
-
+var message = "This is a string";
+ var messageType = typeof message;
 console.log(message);
+console.log(messageType);
+
```

Comment:
can you know type of a variable without a definition 
console.log(typeof message);